scripts/ci/pre_commit_breeze_cmd_line.sh

```diff
@@ -29,9 +28,23 @@ echo "
 .. code-block:: text
 " >"${TMP_FILE}"
 
-export SEPARATOR_WIDTH=80
-export AIRFLOW_CI_SILENT="true"
-./breeze --help | sed 's/^/  /' | sed 's/ *$//' >>"${TMP_FILE}"
+export MAX_SCREEN_WIDTH=100
```

Comment:
110 as that is the line length we have flake8 configured to?

Reply:
That too. But it's more an aesthetic thing :). The output looks nicer with it.